libs/keycodes_ascii/src/lib.rs

```diff
@@ -133,6 +130,7 @@ impl KeyEvent {
     }
 }
 
+// FIXME: this is only true for scancode set 1
```

Comment:
let's discuss this based on my other question about unconditionally setting scancode Set 2. Shall we change this to be compatible with Set 2? Seems like a pretty important consideration.

Reply:
I will have to read up on it. Set 2 can get converted to Set 1 by the PS/2 controller, I'm a bit confused about what it's doing in our setup.